crates/oxc_linter/src/snapshots/eslint_no_array_constructor.snap

```diff
@@ -36,6 +43,20 @@ source: crates/oxc_linter/src/tester.rs
    ╰────
   help: Use an array literal instead
 
+  ⚠ eslint(no-array-constructor): Do not use `new` to create arrays
+   ╭─[no_array_constructor.tsx:1:1]
+ 1 │ new Array(...args);
+   · ──────────────────
+   ╰────
+  help: Use an array literal instead
+
+  ⚠ eslint(no-array-constructor): Do not use `new` to create arrays
+   ╭─[no_array_constructor.tsx:1:1]
+ 1 │ Array(x, y)
```

Comment:
Not sure why I am getting duplicate snapshots here. Must be the diagnostic being reported multiple times when it shouldn't be.

Reply:
The error message is off btw, there is no `new` in the code.

Reply:
Good catch! Thanks.